views/editorial.ejs

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+	<title>Editorials | CodeManics</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" type="text/css" href="/css/sheet.css">
+	<link rel="stylesheet" type="text/css" href="/css/contest_admin.css">
+	<link href='https://fonts.googleapis.com/css?family=Sofia' rel='stylesheet'>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	<link rel="stylesheet" type="text/css" href="/css/grid.css">
+
+</head>
+
+<body>
+	<header>
+		<% include partials/header %>
+    </header>
+    <div class="grid-content content">
+        <div class="box">
+            <h1 class="title center"> Editorial for Question <%= questionNo %> </h1>
```

Comment:
@pranshukharkwal It would be really great if you can show the Question Name on the page as well.

Reply:
@dheeraj135 Done
Looks like this now
![Screenshot from 2020-11-19 01-02-07](https://user-images.githubusercontent.com/21126219/99578447-deba8900-2a02-11eb-9b31-dfb1bd1e918a.png)